src/ol/render/canvas/canvasreplay.js

```diff
@@ -387,6 +387,7 @@ ol.render.canvas.Replay.prototype.replay_ = function(
                 goog.vec.Mat4.getElement(localTransform, 0, 3),
                 goog.vec.Mat4.getElement(localTransform, 1, 3));
           }
+          /*
```

Comment:
We shouldn't leave commented code blocks in the library.  They are sources of mystery to future readers.